ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineGetPayload.java

```diff
@@ -61,6 +76,11 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext request) {
     if (blockWithReceipts.isPresent()) {
       final var proposal = blockWithReceipts.get();
       LOG.atDebug().setMessage("assembledBlock {}").addArgument(() -> proposal).log();
+      ValidationResult<RpcErrorType> forkValidationResult =
+          validateForkSupported(proposal.getHeader().getTimestamp());
+      if (!forkValidationResult.isValid()) {
+        return new JsonRpcSuccessResponse(request.getRequest().getId(), forkValidationResult);
```

Comment:
should this be an error response?

Reply:
You'd think so, but no. Those are reserved for JsonRPCprotocol type errors.